lib/addons/aws-for-fluent-bit/index.ts

```diff
@@ -0,0 +1,73 @@
+import { PolicyStatement } from '@aws-cdk/aws-iam';
+import { Construct } from "@aws-cdk/core";
+
+import { HelmAddOn, HelmAddOnUserProps } from "../helm-addon";
+import { ClusterInfo } from "../../spi/types";
+import { createNamespace } from "../../utils/namespace-utils";
+
+/**
+ * Configuration options for the FluentBit add-on.
+ */
+export interface AwsForFluentBitAddOnProps extends HelmAddOnUserProps {
+    /**
+     * Iam policies for the add-on.
+     */
+    iamPolicies?: PolicyStatement[]
+}
+/**
+ * Default props for the add-on.
+ */
+const defaultProps: AwsForFluentBitAddOnProps = {
+    name: 'fluent-bit',
+    chart: 'aws-for-fluent-bit',
+    release: "ssp-addon-aws-for-fluent-bit",
+    version: '0.1.11',
+    repository: 'https://aws.github.io/eks-charts',
+    namespace: 'kube-system',
+    values: {}
+}
+
+/**
+ * AwsForFluentBitAddOn deploys FluentBit into an EKS cluster using the `aws-for-fluent-bit` Helm chart.
+ * https://github.com/aws/eks-charts/tree/master/stable/aws-for-fluent-bit
+ */
+export class AwsForFluentBitAddOn extends HelmAddOn {
+
+    readonly options: AwsForFluentBitAddOnProps
+
+    constructor(props?: AwsForFluentBitAddOnProps) {
+        super({ ...defaultProps as any, ...props })
+        this.options = this.props
+    }
+
+    deploy(clusterInfo: ClusterInfo): Promise<Construct> {
+        const cluster = clusterInfo.cluster;
+
+        // Create the FluentBit namespace.
+        const namespace = this.options.namespace
+        createNamespace(this.options.namespace!, cluster)
```

Comment:
The CI run caught a failure here: if namespace is left as kube-system this call fails. One way is to check for namespace name, but we could also add a flag, like in VeleroAddOn `createNamespace: boolean` to make it explicit. And that one could false by default.

Reply:
Shouldn't the helper function be updated to do a no op if namespace exists?

Reply:
Good point! The createnamespace function should be leveraging kubetl apply. In CDK it is overwrite flag:

```
ew KubernetesManifest(cluster.stack, name, {
            cluster,
            manifest,
            overwrite: true
        })
```